nrfx/mdk/nrf5340_application_bitfields.h

```diff
@@ -10837,12 +10805,6 @@ POSSIBILITY OF SUCH DAMAGE.
 /* Register: RESET_RESETREAS */
 /* Description: Reset reason */
 
-/* Bit 27 : Reset from network CTRL-AP detected */
-#define RESET_RESETREAS_LCTRLAP_Pos (27UL) /*!< Position of LCTRLAP field. */
-#define RESET_RESETREAS_LCTRLAP_Msk (0x1UL << RESET_RESETREAS_LCTRLAP_Pos) /*!< Bit mask of LCTRLAP field. */
```

Comment:
Another reason to update `hal/nrf_reset.h`, and `helpers/nrfx_reset_reason.h` as well.